source/Generic/SplashScreen/SplashScreenSettingsView.xaml

```diff
@@ -23,13 +23,15 @@
                 <StackPanel Margin="40,0,0,0" IsEnabled="{Binding ElementName=ExecuteDesktopMode, Path=IsChecked}">
                     <CheckBox IsChecked="{Binding Settings.ViewImageSplashscreenDesktopMode}" Content="{DynamicResource LOCSplashScreen_SettingCBviewImageSplashscreenDesktopMode}" Margin="0,10,0,0" />
                     <CheckBox IsChecked="{Binding Settings.ViewVideoDesktopMode}" Content="{DynamicResource LOCSplashScreen_SettingCBviewVideoDesktopMode}" Margin="0,10,0,0" />
+                    <CheckBox IsChecked="{Binding Settings.UseMicroTrailersInDesktopMode}" Content="{DynamicResource LOCSplashScreen_SettingCBuseMicroTrailerDesktopMode}" Margin="0,10,0,0" />
```

Comment:
Some things:

1. Bind their IsEnabled behavior to their parent CheckBox to enable the video functionality
2. Add a left margin of 20 to visually indicate that it's a child element of the parent option
3. A single `Use MicroTrailers` localization string will suffice for both. It will be clear for which mode it is due to their parent option of each checkbox

Reply:
Added name to parent & bound child enabled status to parent and applied other recommended changes.